### PR TITLE
Fixes #372

### LINKED ISSUE #372: Clicking "start tour" opens the tour window twice
The tour window flashes twice when it opens. It should only open once.

### PR DESCRIPTION
**Pull request by idogi**

### CODE REVIEW

**Comment by danielhers** [comment]
Thanks. It's great that you say which issue you fixed, but please also give a brief description of what you did. 

Did you test that all the tour stages still work properly? I remember some of them not showing when I tried fixing this.

**Comment by idogi** [comment]
It seems that the bug stemmed from an unnecessary tour.restart() call.
Removed it and changed the following start() to a restart().

Al stages were tested to appear in order.
 
Tour still has some issues. see #393.

File: static/js/tour.js [hunk 1]
```diff
@@ -95,12 +95,12 @@ function tourClick(){
     }
     checkStepsAdd = false;
     checkStepsAdd2 = false;
-    tour.restart();
+    //tour.restart();
```
Comment (omerxx) [inline static/js/tour.js]: Is the comment necessary?

Comment (danielhers) [inline static/js/tour.js]: Probably not.

**Comment by danielhers** [comment]
@idogi, could you make that description the commit message by doing commit --amend and push -f?

**Comment by omerxx** [comment]
We can also change it inline here directly once it's merged..

**Comment by omerxx** [comment]
I deleted the comment manually

**Pull request merged**